lib/price_calculator.rb

```diff
@@ -0,0 +1,37 @@
+require "terminal-table"
+
+require_relative './cart'
+require_relative './invoice'
+
+class PriceCalculator 
+
+    attr_reader :items
+
+    def initialize
+        @cart = Cart.new.items_bought_list
+        @items_list = []
+        @total_sale_price = 0
+        @total_actual_price = 0
+        calculate_price
+        get_invoice
+    end
+
+
+    def calculate_price
```

Comment:
One method is supposed to do only one thing. Here, we are not only calculating price, but also items_list. That should have been a separate method.